AnnoDesigner/Constants.cs

```diff
@@ -102,5 +102,10 @@ public static class Constants
         /// Used to prevent endless loop on updates.
         /// </summary>
         public const string Argument_Ask_For_Admin = "-askAdmin";
+
+        /// <summary>
+        /// The default number for maximum recent files saved.
```

Comment:
Maybe a better comment here should `The default number of recent files to show`?
And the property - `DefaultRecentFilesToShow`?

Reply:
`DefaultRecentFilesToShow ` sounds to me like a list of default files, in the meaning of a list of file paths.
Also the value could be changed so it isn't really a default value.